### DISCUSSION TITLE
Simplify beet move behaviour when changing beet's music directory

### DISCUSSION THREAD

**Discussion by cl-parsons** [beetbox/beets#5499] [Ideas]
If I change the "directory" configuration value, and I execute "beet move", each file is moved one by one. It would be simpler to move the whole directory.

**Comment by InvisibleFunction** [comment]
Could you copy paste from your terminal session to show exactly what you mean?

**Comment by snejus** [comment]
I think I see what you mean @cl-parsons: the `move` command is designed to read the query from the command line and then process each of the matching songs one by one. In most cases, there will be just a couple of songs that need moving.

Regarding your use case @cl-parsons, I'd suggest you to manually `mv <old-directory> <new-directory>` and update your configuration accordingly:

```diff
-directory: <old-directory>
+directory: <new-directory>
```
